cli/cmd/utils.go

```diff
@@ -187,7 +187,10 @@ func readKubeconfigFromTerraformState(contextLogger *log.Entry) ([]byte, error)
 	contextLogger.Warn("Kubeconfig file not found in assets directory, pulling kubeconfig from " +
 		"Terraform state, this might be slow. Run 'lokoctl cluster apply' to fix it.")
 
-	c, err := initialize(contextLogger)
+	// TODO: Add Terraform verbose support back.
```

Comment:
I understand this means you can't configure verboseness when reading the kubeconfig from the terraform state.

In that case I think I'm fine with this.